frontend/ListAds.elm

```diff
@@ -17,160 +18,231 @@ import Removal
-          [ H.h1
-            [ A.class "list-ads__header" ]
-            [ H.text "Selaa ilmoituksia" ]
-          ]
-        ]
-      , H.div
-        [ A.class "row list-users__filters" ]
+    H.div []
         [ H.div
-          [ A.class "col-xs-12 col-sm-4" ]
-          [ Common.select "list-users" (LocalViewMessage << ChangeDomainFilter) Domain config.domainOptions model ]
+            [ A.class "container" ]
+            [ H.div
+                [ A.class "row" ]
+                [ H.div
+                    [ A.class "col-sm-12" ]
+                    [ H.h1
+                        [ A.class "list-ads__header" ]
+                        [ H.text "Selaa ilmoituksia" ]
+                    ]
+                ]
+            , H.div
+                [ A.class "row list-users__filters" ]
+                [ H.div
+                    [ A.class "col-xs-12 col-sm-3" ]
+                    [ Common.select "list-users" (LocalViewMessage << ChangeDomainFilter) Domain config.domainOptions model ]
+                , H.div
+                    [ A.class "col-xs-12 col-sm-3" ]
+                    [ Common.select "list-users" (LocalViewMessage << ChangePositionFilter) Position config.positionOptions model ]
+                , H.div
+                    [ A.class "col-xs-12 col-sm-3" ]
+                    [ Common.select "list-users" (LocalViewMessage << ChangeLocationFilter) Location Config.finnishRegions model ]
+                , H.div
+                    [ A.class "col-xs-12 col-sm-3" ]
+                    [ selectSorting "list-users" (LocalViewMessage << ChangeSorting) Config.sortingOptions model ]
+                ]
+            ]
         , H.div
-          [ A.class "col-xs-12 col-sm-4" ]
-          [ Common.select "list-users" (LocalViewMessage << ChangePositionFilter) Position config.positionOptions model ]
-        , H.div
-          [ A.class "col-xs-12 col-sm-4" ]
-          [ Common.select "list-users" (LocalViewMessage << ChangeLocationFilter) Location Config.finnishRegions model ]
+            [ A.class "list-ads__list-background" ]
+            [ H.div
+                [ A.class "container last-row" ]
+                (List.map (Util.localViewMap RemovalMessage) <| viewAds loggedInUserMaybe model.removal model.ads)
+            ]
+        ]
+
+
+selectSorting :
+    String
+    -> (Maybe String -> msg)
+    -> List ( String, String )
+    ->
+        { a
+            | selectedSorting : Maybe String
+        }
+    -> H.Html msg
+selectSorting class toMsg options model =
+    let
+        isSelected option =
+            Just option == model.selectedSorting
+    in
+    H.span
+        [ A.class <| class ++ "__select-container" ]
+        [ H.select
+            [ A.class <| class ++ "__select"
+            , E.on "change"
+                (E.targetValue
+                    |> Json.map
+                        (\str ->
+                            case List.find (\tupl -> Tuple.first tupl == str) options of
```

Comment:
Change to similar implementation as in ListUsers with buttons will get rid of this, but in general prefer records over Tuples, when relevant.